styleguide/templates/styleguide/styleguide.html

```diff
@@ -29,8 +29,14 @@
     <nav id=styleguide-menu>
         <ul>
         {% for name, slug, sub_slugs in templates %}
-            {% url "styleguide_page" slug as path %}
             {% open_menu request.path slug as open_parent %}
+            {% if slug %}
+                {% url "styleguide_page" slug as path %}
+                <li class="{% if path in request.path %}selected {% endif%}{% if open_parent %}children{% endif %}">
+            {% else %}
+                {% url "styleguide" as path %}
+                <li class="{% if path == request.path %}selected {% endif%}">
+            {% endif %}
```

Comment:
The 'Introduction' choice on the menu remains highlighted when you navigate to other sections of the site.

Reply:
Also, in related news, there appear to be extra borders that snuck there way into each menu item.

Reply:
I don't see either of these behaviors at all, can we look at it on your machine together?